### ISSUE TITLE
Discrepancies between export and import formats

### ISSUE THREAD

**Issue by pinarpink** [elixir-luxembourg/daisy#188]
**Describe the bug (current behaviour)**

The json export obtained from DAISY does not match the JSONs used for import. These need to be harmonised and made conformant to the ELU json schemas [here](https://git-r3lab.uni.lu/pinar.alper/metadata-tools/-).

**To reproduce**

1.  run `python36 manage.py export_datasets -f ds-export.json`
2. compare `ds-export.json` with `/data/demo/datasets.json`

**Expected behavior**
json dictionaries should be harmonised and be conformant to ELU schema

**Environment:**

 - Deployment: daisy.lcsb.uni.lu
 - Version 1.4.0
**Additional information (error logs, screenshots)**

**Comment by jLebioda** [comment]
This is how the data look when exported from DAISY (`python36 manage.py export_datasets -f ds-export.json`) now:

```
{
    "$schema": "https://git-r3lab.uni.lu/pinar.alper/metadata-tools/raw/master/metadata_tools/resources/elu-dataset.json",
    "items": [
        {
            "source": "example.com",
            "id_at_source": "1",
            "project": "EPIC-DREM",
            "name": "EPIC-DREM Sequencing data",
            "elu_accession": "-",
            "description": null,
            "elu_uuid": "ac32f877-a4b1-4a64-bc02-32ea6e6adb15",
            "other_external_id": null,
            "data_declarations": [
                {
                    "title": "ChIP-Seq, RNA-seq, ...",
                    "data_types": [
                        "Samples",
                        "Whole_genome_sequencing",
                        "RNASeq"
                    ],
                    "access_category": "controlled_access",
                    "subjects_category": "controls",
                    "de_identification": "pseudonymization",
                    "consent_status": "homogeneous",
                    "has_special_subjects": false,
                    "special_subjects_description": "",
                    "embargo_date": null,
                    "storage_end_date": null,
                    "storage_duration_criteria": null
                }
            ],
            "storages": [
                {
                    "platform": "master",
                    "location": "https://webdav-r3lab.uni.lu/",
                    "accesses": [
                        "The data is available on the sequencing platform's webdav server."
                    ]
                }
            ],
            "transfers": [],
            "contacts": [
                {
                    "first_name": "Alice",
                    "last_name": "White",
                    "email": "alice.white@uni.lu",
                    "role": "Researcher",
                    "affiliations": [
                        "Luxembourg Centre for Systems Biomedicine"
                    ]
                },
                {
                    "first_name": "John",
                    "last_name": "Black",
                    "email": "john.black@uni.lu",
                    "role": "Researcher",
                    "affiliations": [
                        "Luxembourg Centre for Systems Biomedicine"
                    ]
                }
            ]
        },
        {
            "source": "example.com",
            "id_at_source": "2",
            "project": "Iron-deficiency",
            "name": "Iron-deficiency dataset",
            "elu_accession": "-",
            "description": null,
            "elu_uuid": "c99513f5-5c51-4327-8941-efc2d9a1fc5d",
            "other_external_id": null,
            "data_declarations": [
                {
                    "title": "PD, small scale integrity data (german cohort)",
                    "data_types": [
                        "Samples",
                        "Other"
                    ],
                    "access_category": null,
                    "subjects_category": "cases_and_controls",
                    "de_identification": "pseudonymization",
                    "consent_status": "unknown",
                    "has_special_subjects": false,
                    "special_subjects_description": null,
                    "embargo_date": null,
                    "storage_end_date": null,
                    "storage_duration_criteria": null
                }
            ],
            "storages": [],
            "transfers": [],
            "contacts": []
        }
    ]
}
```


This is how the example data (`$REPO/data/demo/datasets.json`) looks like:
[
    {
        "project": "EPIC-DREM",
        "name": "EPIC-DREM Sequencing data",
        "description": null,
        "data_declarations": [
            {
                "title": "ChIP-Seq, RNA-seq, ...",
                "data_types": [
                    "RNASeq",
                    "Samples",
                    "Whole_genome_sequencing"
                ],
                "data_type_notes": "Chip-seq  time series data",
                "access_category": "controlled_access",
                "subjects_category": "controls",
                "de_identification": "pseudonymization",
                "consent_status": "homogeneous",
                "special_subjects_description": "",
                "embargo_date": null,
                "storage_end_date": "2021-04-05",
                "storage_duration_criteria": "The project PI will assess whether data should be retained for further research at the storage end date.",
                "has_special_subjects": false,
                "consent_status": "homogeneous",
                "use_restrictions": [
                    {
                        "ga4gh_code": "COL-[XX]",
                        "note": "Data is shared as part of the LUX-Epigen programme, a collaboration with CHL, IBBL, LCSB and LSRU."
                    },
                    {
                        "ga4gh_code": "DS-[XX](CC)",
                        "note": "Data is consented for cancer research studies only."
                    },
                    {
                        "ga4gh_code": "GS-[XX]",
                        "note": "Data is consented for sharing outside EU/EEA region."
                    }
                ]
            }
        ],
        "storages": [
            {
                "platform": "application",
                "locations": [
                    "https://webdav-r3lab.uni.lu/"
                ],
                "category": "master",
                "accesses": [
                    "The data is available on the sequencing platform's webdav server."
                ]
            }
        ],
        "transfers": [],
        "contacts": [
            {
                "first_name": "Alice",
                "last_name": "White",
                "email": "alice.white@uni.lu",
                "role": "Principal_Investigator",
                "institution": "ELU_I_77"
            },
            {
                "first_name": "John",
                "last_name": "Black",
                "email": "john.black@uni.lu",
                "role": "Principal_Investigator",
                "institution": "ELU_I_77"
            }
        ]
    },
    {
        "project": "Iron-deficiency",
        "name": "Iron-deficiency dataset",
        "data_declarations": [{
            "title": "PD, small scale integrity data (german cohort)",
            "data_type_notes": "Small-scale protein, mRNA or mtDNA integrity data, cell and mitochondrial function data\n",
            "data_types": [
                "Other",
                "Samples"
            ],
            "de_identification": "pseudonymization",
            "subjects_category": "cases_and_controls",
            "has_special_subjects": false,
            "source_notes": "Data is from collaborator.",
            "storage_end_date": "2022-02-28",
            "storage_duration_criteria": "The project PI will assess whether data should be retained for further research at the storage end date.",
            "use_restrictions": [
                {
                    "ga4gh_code": "PUB",
                    "note": "Acknowledgement required."
                },
                {
                    "ga4gh_code": "TS-[XX]",
                    "note": "Data is obtained for a limited duration. 2022-02-28"
                }
            ]
        }
        ]
    }
]
```

**Comment by jLebioda** [comment]
While the general idea of the issue is pretty clear, I'd like just to double check. 

The format of the exported file appears more up-to-the-date than the import counterpart. Is the issue mostly about updating the _importer_ and the file used to load example data?

**Comment by pinarpink** [comment]
Indeed this is an issue to do with extending the importer and making sure entities in the schemas [here](https://git-r3lab.uni.lu/pinar.alper/metadata-tools/-/tree/master/metadata_tools/resources) can be imported. Specifically the `submission` entity.

**Comment by pinarpink** [comment]
This is addressed in release 1.6.0, closing
